.github/workflows/linux-tests.yml

```
@@ -0,0 +1,60 @@
+name: Linux tests
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  linux:
+    name: Linux Py${{ matrix.PYTHON_VERSION }}
+    runs-on: ubuntu-latest
+    env:
+      CI: True
+      PYTHON_VERSION: ${{ matrix.PYTHON_VERSION }}
+      RUNNER_OS: 'ubuntu'
+    strategy:
+      fail-fast: false 
+      matrix:
+        PYTHON_VERSION: ['3.7', '3.8', '3.9']
```

Lets change this to use 3.8 and 3.10 only:
```suggestion
        PYTHON_VERSION: ['3.8', '3.10']
```